Makefile

```diff
@@ -212,7 +212,7 @@ EXTRA_DOCKER_ARGS	+= -e GO111MODULE=on
 
 # Allow libcalico-go and the ssh auth sock to be mapped into the build container.
 ifdef LIBCALICOGO_PATH
-  EXTRA_DOCKER_ARGS += -v $(LIBCALICOGO_PATH):/github.com/projectcalico/libcalico-go:ro
+  EXTRA_DOCKER_ARGS += -v $(LIBCALICOGO_PATH):/go/src/github.com/projectcalico/libcalico-go:ro
```

Comment:
Please use `-v ....:/go/src/$(PACKAGE_NAME):rw \` to follow what the rest of the file does.

Reply:
I may be missing something but PACKAGE_NAME name is github.com/projectcalico/felix, and it needs to be github.com/projectcalico/libcalico-go here

Reply:
Ah, my bad.. was looking at wrong repo. LGTM